testcases/open_posix_testsuite/conformance/interfaces/mq_timedsend/12-1.c

```diff
@@ -148,12 +148,13 @@ int main(void)
 	if (ret != 0 && ret != PTHREAD_BARRIER_SERIAL_THREAD)
 		error_and_exit(ret, "pthread_barrier_wait start");
 
+	struct timespec completion_wait_ts = {0, SIGNAL_DELAY_MS*1000};
```

Comment:
This should be "* 1000 * 1000".

Reply:
Actually, it should be *1000. 1 us (e-6) == 1000 ns (e-9)

Reply:
Oh shoot. You're right :D. Fixing...